BUILDING.md

```diff
@@ -87,7 +87,12 @@ To test if Node.js was built correctly:
 $ node -e "console.log('Hello from Node.js ' + process.version)"
 ```
 
-### Android / Android based devices, aka. Firefox OS
+### Android / Android-based devices (e.g., Firefox OS)
+
+Although these instructions for building on Android are provided, please note
+that Android is not a supported platform at this time. Patches to improve
```

Comment:
This is a small nit, but perhaps it should say `is not an officially supported platform`?